src/DependencyInjection/DoctrineExtension.php

```diff
@@ -196,6 +197,9 @@ protected function dbalLoad(array $config, ContainerBuilder $container)
         $connWithLogging   = [];
         $connWithProfiling = [];
         $connWithBacktrace = [];
+        $ttlByConnection   = [];
+        $connWithTtl       = [];
```

Comment:
It's not needed to have 2 variables for this. Wherever you use `$connWithTtl` you can just do `array_keys($ttlByConnection)` instead

Reply:
Indeed, I'll update it. thanks for your feedback